README.md

```diff
@@ -33,6 +33,8 @@ Check out the [the examples directory](examples) folder for more examples on how
 
 Read the [contributing guide](CONTRIBUTING.md) to learn about our development process, reporting bugs and improvements, creating issues and asking questions.
 
+check out the [developing process](docs/development/developing.md) for help
+
```

Comment:
Hey! Thanks for noticing we're not linking that here. I have a few remarks:

This could be better phrased as in explaining what the `developing.md` does and how it's supposed to help you.
Grammar nitpicks: it'd be nice if you use capital letters + punctuation.